boards/nucleo_f429zi/src/main.rs

```diff
@@ -291,12 +304,108 @@ pub unsafe fn reset_handler() {
+        stm32f4xx::gpio::PIN[1][0].as_ref().unwrap(), //D33
+        stm32f4xx::gpio::PIN[4][0].as_ref().unwrap(), //D34
+        stm32f4xx::gpio::PIN[1][11].as_ref().unwrap(), //D35
+        stm32f4xx::gpio::PIN[1][10].as_ref().unwrap(), //D36
+        stm32f4xx::gpio::PIN[4][15].as_ref().unwrap(), //D37
+        stm32f4xx::gpio::PIN[4][14].as_ref().unwrap(), //D38
+        stm32f4xx::gpio::PIN[4][12].as_ref().unwrap(), //D39
+        stm32f4xx::gpio::PIN[4][10].as_ref().unwrap(), //D40
+        stm32f4xx::gpio::PIN[4][7].as_ref().unwrap(), //D41
+        stm32f4xx::gpio::PIN[4][8].as_ref().unwrap(), //D42
+        // SDMMC
+        stm32f4xx::gpio::PIN[2][8].as_ref().unwrap(), //D43
+        stm32f4xx::gpio::PIN[2][9].as_ref().unwrap(), //D44
+        stm32f4xx::gpio::PIN[2][10].as_ref().unwrap(), //D45
+        stm32f4xx::gpio::PIN[2][11].as_ref().unwrap(), //D46
+        stm32f4xx::gpio::PIN[2][12].as_ref().unwrap(), //D47
+        stm32f4xx::gpio::PIN[3][2].as_ref().unwrap(), //D48
+        stm32f4xx::gpio::PIN[6][2].as_ref().unwrap(), //D49
+        stm32f4xx::gpio::PIN[6][3].as_ref().unwrap(), //D50
+        // USART
+        stm32f4xx::gpio::PIN[3][7].as_ref().unwrap(), //D51
+        stm32f4xx::gpio::PIN[3][6].as_ref().unwrap(), //D52
+        stm32f4xx::gpio::PIN[3][5].as_ref().unwrap(), //D53
+        stm32f4xx::gpio::PIN[3][4].as_ref().unwrap(), //D54
+        stm32f4xx::gpio::PIN[3][3].as_ref().unwrap(), //D55
+        stm32f4xx::gpio::PIN[4][2].as_ref().unwrap(), //D56
+        stm32f4xx::gpio::PIN[4][4].as_ref().unwrap(), //D57
+        stm32f4xx::gpio::PIN[4][5].as_ref().unwrap(), //D58
+        stm32f4xx::gpio::PIN[4][6].as_ref().unwrap(), //D59
+        stm32f4xx::gpio::PIN[4][3].as_ref().unwrap(), //D60
+        stm32f4xx::gpio::PIN[5][8].as_ref().unwrap(), //D61
+        stm32f4xx::gpio::PIN[5][7].as_ref().unwrap(), //D62
+        stm32f4xx::gpio::PIN[5][9].as_ref().unwrap(), //D63
+        stm32f4xx::gpio::PIN[6][1].as_ref().unwrap(), //D64
+        stm32f4xx::gpio::PIN[6][0].as_ref().unwrap(), //D65
+        stm32f4xx::gpio::PIN[3][1].as_ref().unwrap(), //D66
+        stm32f4xx::gpio::PIN[3][0].as_ref().unwrap(), //D67
+        stm32f4xx::gpio::PIN[5][0].as_ref().unwrap(), //D68
+        stm32f4xx::gpio::PIN[5][1].as_ref().unwrap(), //D69
+        stm32f4xx::gpio::PIN[5][2].as_ref().unwrap(), //D70
+        stm32f4xx::gpio::PIN[0][7].as_ref().unwrap(), //D71
+        // ADC Pins
+        stm32f4xx::gpio::PIN[0][3].as_ref().unwrap(), //A0
+        stm32f4xx::gpio::PIN[2][0].as_ref().unwrap(), //A1
+        stm32f4xx::gpio::PIN[2][3].as_ref().unwrap(), //A2
+        stm32f4xx::gpio::PIN[5][3].as_ref().unwrap(), //A3
+        stm32f4xx::gpio::PIN[5][5].as_ref().unwrap(), //A4
+        stm32f4xx::gpio::PIN[5][10].as_ref().unwrap(), //A5
+        stm32f4xx::gpio::PIN[1][1].as_ref().unwrap(), //A6
+        stm32f4xx::gpio::PIN[2][2].as_ref().unwrap(), //A7
+        stm32f4xx::gpio::PIN[5][4].as_ref().unwrap()  //A8
```

Comment:
I don't think we should export the ADC pins as GPIO.

Reply:
It seems that some of the ADC pins are available as GPIO. Should we remove the export completely or just add a comment to instruct users to command out the ADC pins?

```suggestion
        // ADC Pins - comment these lines if you want to use the ADC Inputs
        stm32f4xx::gpio::PIN[0][3].as_ref().unwrap(), //A0
        stm32f4xx::gpio::PIN[2][0].as_ref().unwrap(), //A1
        stm32f4xx::gpio::PIN[2][3].as_ref().unwrap(), //A2
        stm32f4xx::gpio::PIN[5][3].as_ref().unwrap(), //A3
        stm32f4xx::gpio::PIN[5][5].as_ref().unwrap(), //A4
        stm32f4xx::gpio::PIN[5][10].as_ref().unwrap(), //A5
        stm32f4xx::gpio::PIN[1][1].as_ref().unwrap(), //A6
        stm32f4xx::gpio::PIN[2][2].as_ref().unwrap(), //A7
        stm32f4xx::gpio::PIN[5][4].as_ref().unwrap()  //A8
```

Reply:
I think we should export them in the example board the way they are marked on the physical board. Users shouldn't have to know somehow that pins labeled analog are actually by default just GPIO.

Reply:
I commented out the A ports and placed a comment to enable them if needed.

What do you mean by "export them in the example board the way they are marked on the physical board"? When using them from libtock-c, they appear as numbers, maybe I am missing something.

Reply:
I'm referring to the silkscreen on the PCB.

Reply:
Pins are labeled from D0 to D15, and from A0 - A5. All the others have alternate functions displayed. When looking at the manual (paper on the back), pins are displayed D0 to D71 and A0 to A8.

Should we add the P{PORT}{pin} next to the D and A comments?